.docker/mariadb/conf/config.cfg

```diff
@@ -1,2 +1 @@
 default_authentication_plugin=mysql_native_password
```

Comment:
Did you made a test removing this?
If work fine without this, we can customize mounting the config file as volume inside the container, but at this case, using a file from `volumes/mariadb/config` by example because is a file that isn't versioned. Maybe this is a edge case and we can remove this logic if work fine without this row. Another motivation is because I didn't found the place that this config is used.

Reply:
I've passed it for .env and the things is working fine.